src/pages/dho/Configuration.vue

```diff
@@ -265,7 +265,7 @@ export default {
             communityVotingEnabled: form.communityVotingEnabled ? 1 : 0,
             upvoteHeadDelegateRound: form.upvoteHeadDelegateRound ? 1 : 0,
 
-            upvoteStartDateTime: new Date(`${upvoteStartDate.replace(/\//g, '-')}T${upvoteStartTime}:00.000${timezoneHours}:${timezoneMinutes}`).toISOString(),
+            upvoteStartDateTime: new Date(`${upvoteStartDate.replace(/\//g, '-')}T${upvoteStartTime}:00.000${timezoneHours}:${timezoneMinutes}`).toISOString().replace('Z', ''),
```

Comment:
Missing upvoteStartDate would break the app. Ensure that it is present or use optional chaining.  In general this formatting should be extracted into utility function and reused wherever we need to format the date in the same way.